### PR TITLE
Improves unittest CC for azure_controller_vmss

### PR DESCRIPTION
**Pull request by weijiehu**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

/kind cleanup

**What this PR does / why we need it**:
Improves unit test code coverage for azure_controller_vmss

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

/sig cloud-provider
/area provider/azure
/priority important-soon

### CODE REVIEW

File: staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_vmss_test.go [hunk 1]
```diff
@@ -0,0 +1,324 @@
+// +build !providerless
+
+/*
+Copyright 2019 The Kubernetes Authors.
```
Comment (feiskyer) [inline staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_vmss_test.go:4]: s/2019/2020/

File: staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_vmss_test.go [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+	"k8s.io/legacy-cloud-providers/azure/clients/vmssclient/mockvmssclient"
+	"k8s.io/legacy-cloud-providers/azure/clients/vmssvmclient/mockvmssvmclient"
+	"k8s.io/legacy-cloud-providers/azure/retry"
+)
+
+func TestAttachDiskWithVMSS(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	fakeStatusNotFoundVMSSName := types.NodeName("FakeStatusNotFound")
```
Comment (feiskyer) [inline staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_vmss_test.go:43]: s/FakeStatusNotFound/FakeStatusNotFoundNodeName/ for clear

File: staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_vmss_test.go [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+			vmssvmName:     "vmss00-vm-000000",
+			isManagedDisk:  false,
+			existedDisk:    compute.Disk{Name: to.StringPtr("disk-name")},
+			expectedErr:    true,
+			expectedErrMsg: fmt.Errorf("Retriable: false, RetryAfter: 0s, HTTPStatusCode: 404, RawError: instance not found"),
+		},
+	}
+
+	for i, test := range testCases {
+		fmt.Printf("[%s]\r\n", test.desc)
```
Comment (feiskyer) [inline staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_vmss_test.go:96]: please remove "fmt.Printf" lines

File: staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_vmss_test.go [hunk 4]
```diff
@@ -0,0 +1,324 @@
...
+		assert.Equal(t, test.expectedErr, err != nil, "TestCase[%d]: %s, return error: %v", i, test.desc, err)
+		assert.Equal(t, test.expectedErrMsg, err, "TestCase[%d]: %s, expected error: %v, return error: %v", i, test.desc, test.expectedErrMsg, err)
+	}
+}
+
+func TestDetachDiskWithVMSS(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	fakeStatusNotFoundVMSSName := types.NodeName("FakeStatusNotFound")
```
Comment (feiskyer) [inline staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_vmss_test.go:144]: s/"FakeStatusNotFound"/"FakeStatusNotFoundVMSSName"

File: staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_vmss_test.go [hunk 5]
```diff
@@ -0,0 +1,324 @@
...
+			vmssVMList:  []string{"vmss00-vm-000000", "vmss00-vm-000001", "vmss00-vm-000002"},
+			vmssName:    "vmss00",
+			vmssvmName:  "vmss00-vm-000000",
+			existedDisk: compute.Disk{Name: to.StringPtr("disk-name-err")},
+			expectedErr: false,
+		},
+	}
+
+	for i, test := range testCases {
+		fmt.Printf("[%s]\r\n", test.desc)
```
Comment (feiskyer) [inline staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_vmss_test.go:192]: remove fmt line

**Review by feiskyer — COMMENTED** [review]
/ok-to-test

**Comment by weijiehu** [comment]
@feiskyer I have fixed all commented issues.

**Review by feiskyer — COMMENTED** [review]
/lgtm
/approve
/retest

**Pull request merged**